charts/tidb-cluster/templates/tidb-initializer-job.yaml

```diff
@@ -19,6 +19,31 @@ spec:
         app.kubernetes.io/component: tidb-initializer
     spec:
       restartPolicy: OnFailure
+      initContainers:
+      # Before procedding to initialize TiDB, we wait for the TiDB host:port is
+      # open. This avoids unnecessary pod restarts which may result in a very long
+      # restart delay (10s, 20s, .., 5 mins).
+      # https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#restart-policy
+      - name: wait
+        image: {{ .Values.busybox.image }}
+        imagePullPolicy: {{ .Values.busybox.imagePullPolicy | default "IfNotPresent" }}
+        command:
+        - sh
+        - -c
+        - |
+          trap exit TERM
```

Comment:
able to terminate fast on kill container signal (TERMINAL) (xref: https://github.com/kubernetes/kubernetes/pull/64015#issue-188961717)

Reply:
I always run something like [dumb-init](https://github.com/Yelp/dumb-init) in my pods so that these issues don't show up